docs/en/your-profile-and-preferences/customize-group-chat-notifications.md

```diff
@@ -8,9 +8,65 @@ hide:
 
 # Customize group chat notifications
 
- :octicons-tools-24: In progress
+You can customize the messages and mentions notifications you receive in group chats. By doing so, you only receive the group chat notifications relevant to you.
```

Comment:
You can choose which message and mention notifications you want to receive for any group chat. This helps you only see those notifications that are relevant to you.